frontend/src/Components/HomePage/UpcomingTrips/TripDetails.css

```diff
@@ -227,6 +227,25 @@ p {
     color: #1a73e8;
 }
 
+.icons {
+    display: flex;
+    align-items: center;
+}
+
+.icons svg {
+    font-size: 20px;
+    color: #333;
+}
+
+.seat-bids-content {
+    padding: 10px 0;
+    border-bottom: 1px solid #e0e0e0;
+}
+
+.seat-bids-content p {
+    margin: 0;
```

Comment:
Would margin be zero by default, one recommendation is [BEM](https://getbem.com/naming/) to make css cleaner, don't worry about it here though.